src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -0,0 +1,49 @@
+package com.google.sps.data;
+
+import com.google.api.services.compute.model.Operation;
+import java.io.IOException;
+import java.util.Optional;
+
+/** Interface for the ComputerAPIClient class. */
+public interface ComputeAPIClientInterface {
+  /** 
+   * Creates an environment, calls each operation and tracks to see if they 
+   * are executed.
+   * @param {String} operation - the name of the method that should be called.
+   * @param {String} instanceName - the name of the instance being operated on.
+   * @param {Optional<String>} password - the password needed to access this 
+   *    instance. Not all operations requires a password.
+   */
+  void helper(String operation, String instanceName, 
+    Optional<String> password);
+
+  /** 
+   * Creates a new Instance.
+   * @param {String} password - the password needed to access this instance. 
```

Comment:
What do you mean access the instance? Do you the VNC server password? If that's the case, make that explicit. And come up with a better name than "password" too. It's a bit too generic

Reply:
I changed the param to sessionId since the startup script already handles the conversion from id to vnc server password

Reply:
I think maybe you're making it harder than it needs to be? Just rename the variable to vncServerPassword and change teh wording in the documentation. "the password needed to access this instance" is too generic and is not a good description. That could literaly mean anything. I jst meant be very specific. Access to what? In this case, it's access to teh vnc server. 

Unless im missing something. It is late and my brain is fried